client/src/components/resources/ManageResourceCard.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Box, Text, Anchor } from 'grommet'
+import Link from 'next/link'
+import { ResourceCard } from './ResourceCard'
+
+export const ManageResourceCard = ({ resource }) => {
+  // How do we get pending requests
+  // What do we link to when there are pending requests
+  const editLink = `/account/manage-resources/${resource.id}`
+  const teamLink = `/account/teams/${resource.organization.id}`
+  const viewLink = `/resources/${resource.id}`
+
+  // modal for deleting a resource?
```

Comment:
Could you make issues out of this and the above comment? It's a bit hard to keep track of in this form.

Reply:
Thats ok as I flush this out I will end up making that modal